sdk/tables/data-tables/review/data-tables.api.md

```diff
@@ -304,11 +304,43 @@ export interface SignedIdentifier {
     id: string;
 }
 
+// @public
+export interface TableBatch {
+    createEntities: <T extends object>(entitites: TableEntity<T>[]) => void;
+    createEntity: <T extends object>(entity: TableEntity<T>) => void;
+    deleteEntity: (partitionKey: string, rowKey: string, options?: DeleteTableEntityOptions) => void;
+    partitionKey: string;
+    submitBatch: () => Promise<TableBatchResponse>;
+    updateEntity: <T extends object>(entity: TableEntity<T>, mode: UpdateMode, options?: UpdateTableEntityOptions) => void;
+}
+
+// @public
+export interface TableBatchEntityResponse {
+    body?: any;
+    etag?: string;
+    rowKey?: string;
+    status: number;
+}
+
+// @public
+export interface TableBatchOptions {
```

Comment:
Are there cases where folks would want to explicitly set these values, or is this simply because we need to do it for determinism in our tests with respect to the recordings?  If this is just for testing, I do wonder if there's some way for us to hide it, but to be honest, I don't know what that would look like.  Maybe Jeff has some idea.

Reply:
We should be able to make the recorder smart enough to not require these if they are indeed just for testing.

Reply:
I was able to get rid of these by stubbing generateUuid in the tests